src/app/schedule/schedule.component.ts

```diff
@@ -21,12 +20,20 @@ export class ScheduleComponent implements OnInit {
   ngOnInit() {
```

Comment:
remove empty init

Reply:
Removing ngOnInit in the component causes an error